apps/modernization-ui/src/apps/patient/add/extended/AddPatientExtendedForm.tsx

```diff
@@ -47,7 +47,7 @@ export const AddPatientExtendedForm = () => {
     });
 
     return (
-        <>
+        <div className={styles.addPatientFormContainer}>
```

Comment:
Question: What problem does this solve?

Reply:
It is not needed I was just testing in the parent component but will get rid of this